cranelift/filetests/src/test_zkasm.rs

```diff
@@ -237,6 +240,62 @@ mod tests {
         program.join("\n")
     }
 
+    fn run_wat_file(path: &Path) -> Result<(), Box<dyn std::error::Error>> {
+        let engine = Engine::default();
+        let wat = std::fs::read_to_string(path).unwrap();
+        let pat = r#"(import "env" "assert_eq" (func $assert_eq (param i32) (param i32)))"#;
+        let assert_type = if wat.contains(pat) { 32 } else { 64 };
```

Comment:
If such checks are expected to be done more frequently, it might be worth introducing a helper function based on [wasmtime::Module::imports](https://docs.rs/wasmtime/latest/wasmtime/struct.Module.html#method.imports). Something like:

```rust
fn imports_host_function(module: ..., import_module: ..., name: ...., type: ...) -> bool {
    let imports = module.imports();
    // Check the host function is imported.
    // Docs of Module::imports() give hints on how to use the data.
}
```

Reply:
Thanks a lot, it helps to make assert type recognition much better